### PR TITLE
chore: use wasmbuild instead of _build.ts

### PR DESCRIPTION
**Pull request by littledivy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,27 +215,16 @@ be added as follows:
 ```
 > rustup target add wasm32-unknown-unknown
 > cargo install -f wasm-bindgen-cli
+> deno install --unstable -A -f -n wasmbuild https://raw.githubusercontent.com/denoland/wasmbuild/d51c673a321ecf4f336a467bb1a4a5327e783481/main.ts
```
Comment (kitsonk) [inline README.md:218]: Can we be more precise than `-A`?

Comment (kitsonk) [reply README.md:218]: Also, can we publish on `deno.land/x`?

**Review by kitsonk — COMMENTED** [review]
Was the lib rebuilt with the command and build steps? There seems to be no changes to the `/lib` which I find surprising.

File: README.md [hunk 2]
```diff
@@ -215,27 +215,16 @@ be added as follows:
...
-
-```
-> ./_build.ts
-```
-
-But can be manually invoked like:
-
 ```
-> deno run --unstable _build.ts
+> wasmbuild --no-default-features --features wasm
```
Comment (dsherret) [inline README.md:225]: Until wasmbuild stabilizes we should probably continue using a script with a pegged version (ex. getting https://github.com/denoland/wasmbuild/issues/3 done)

**Comment by kitsonk** [comment]
I really think we need to figure out a solution that works without using `deno install`, or at least if it is `deno install`.  Right now, on OSX and Linux systems, once setup you just do `./_build.ts` and changes to the build or the build dependencies are reflected in the build.

We should also take this opportunity to have CI do the building, but there is still a need to do a local build for testing and validation.

**Comment by dsherret** [comment]
This is resolved in main now.

**Pull request closed without merging**